### ISSUE TITLE
issue with force property backwards compatibility v4 vs. v3

### ISSUE THREAD

**Issue by dplansky** [d3/d3-force#101]
Just starting out with D3.js for network visualization purposes. 
Was testing out some basic graphs that worked with http://d3js.org/d3.v3.min.js  but not with http://d3js.org/d3.v4.min.js

I tested it with the code in http://bl.ocks.org/sathomas/11550728 and http://jsdatav.is/visuals.html?id=11560633

When you use the v3 it works, when you use v4 there is a JS error 
"Uncaught TypeError: Cannot read property 'force' of undefined at <file:line>"

Unless there is some new aspect of the library shouldn't v4 work like v3 for the basics? If I'm missing something please let me know.

**Comment by steveharoz** [comment]
I ran into a similar issue when I was learning. The short answer to your question is the v4 is NOT backwards compatible with v3. The changes needed are minor but serious enough to cause things to break.

There are some basic examples in the [readme](https://github.com/d3/d3-force/blob/master/README.md), and I made [this demo](https://bl.ocks.org/steveharoz/8c3e2524079a8c440df60c1ab72b5d03) to understand how the different forces work.

If you're going to use v4, I recommend avoiding any tutorials with v3. It's just not worth the confusion.
